<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>295</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>295</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>501</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(32,117,218);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>6</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Droid Sans Fallback&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>用户注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="headportrait_show">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>40</y>
     <width>127</width>
     <height>165</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>115</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/uploadhead_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_reg">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>111</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/register.png);
border-style: flat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="upload_headportrait">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>210</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/uploadhead.png);
border-style: flat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>38</y>
     <width>321</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Droid Sans Fallback&quot;;
color: rgb(112, 112, 112);</string>
      </property>
      <property name="text">
       <string>邮        箱：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="ledit_email">
      <property name="styleSheet">
       <string notr="true">border-style:  flat;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);
font: 10pt &quot;Droid Sans Fallback&quot;;</string>
      </property>
      <property name="text">
       <string>用  户  名：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ledit_username">
      <property name="styleSheet">
       <string notr="true">border-style: flat;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);
font: 10pt &quot;Droid Sans Fallback&quot;;</string>
      </property>
      <property name="text">
       <string>密        码：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="ledit_pwd">
      <property name="styleSheet">
       <string notr="true">border-style: flat;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);
font: 10pt &quot;Droid Sans Fallback&quot;;</string>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="ledit_pwdc">
      <property name="styleSheet">
       <string notr="true">border-style: flat;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="Line" name="line_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Droid Sans Fallback&quot;;
color: rgb(112, 112, 112);</string>
      </property>
      <property name="text">
       <string>昵        称：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="ledit_nickname">
      <property name="styleSheet">
       <string notr="true">border-style: flat;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="Line" name="line_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(112, 112, 112);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
